README.md

```diff
@@ -28,30 +90,111 @@ cd comms-gatekeeper
 yarn install
 ```
 
+3. Build the project:
+
+```bash
+yarn build
+```
+
 ### Configuration
 
 The service uses environment variables for configuration. Create a `.env` file in the root directory following the ones defined in the `.env.default` file.
 
-### Starting the Server
+Key configuration variables include:
+
+- `PG_COMPONENT_PSQL_CONNECTION_STRING`: PostgreSQL connection string
+- `LIVEKIT_HOST`, `LIVEKIT_API_KEY`, `LIVEKIT_API_SECRET`: LiveKit configuration
+- `CATALYST_URL`: Catalyst content server URL
+- `PLACES_API_URL`: Places API endpoint
+- `SOCIAL_SERVICE_URL`: Social service endpoint
+- `AWS_SNS_ARN`: SNS topic ARN for event notifications
 
-1. **Start the PostgreSQL database:**
+### Running the Service
+
+#### Setting up the environment
+
+In order to successfully run this server, external dependencies such as databases must be provided.
+
+To do so, this repository provides you with a `docker-compose.yml` file for that purpose. In order to get the environment set up, run:
 
 ```bash
-docker-compose up
+docker-compose up -d
 ```
 
-2. **Start the application:**
+This will start:
+- PostgreSQL database on port `5450`
+
+#### Running in development mode
+
+To run the service in development mode:
 
 ```bash
 yarn start:dev
 ```
 
+This will:
+- Build the TypeScript code
+- Start the server with hot-reload capabilities
+
+For watch mode with automatic rebuilds:
+
+```bash
+yarn dev
+```
+
 ### Database Migrations
 
-The service uses `node-pg-migrate` for database migrations. Check the [documentation](https://salsita.github.io/node-pg-migrate/) for more information.
+The service uses `node-pg-migrate` for database migrations. Migrations are located in `src/migrations/`.
```

Comment:
We don't have anything related to database migrations in the original docs, it's a great idea to add something about it! I'll add it.